composer.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "newfold-labs/wp-module-migration",
+    "autoload": {
+        "psr-4": {
+            "NewfoldLabs\\WP\\Module\\Migration\\": "includes/"
+        },
+        "files": [
+            "bootstrap.php"
+        ]
+    },
+    "authors": [
+        {
+            "name": "Beyond",
+            "email": "team-spartans@newfold.com"
+          }
+    ],
+    "repositories": [
+        {
+          "type": "composer",
+          "url": "https://newfold-labs.github.io/satis/",
+          "only": [
+            "newfold-labs/*"
+          ]
+        },
+        {
+            "type": "vcs",
+            "url": "git@github.com:InstaWP/connect-helpers.git"
+        }
+      ],
+      "require": {
+          "newfold-labs/wp-module-loader": "^1.0.10",
+          "instawp/connect-helpers": "dev-main"
+        },
+      "require-dev": {
+        "newfold-labs/wp-php-standards": "^1.2"
+      },
+      "scripts": {
+        "i18n": [
+          "vendor/bin/wp i18n make-pot . ./languages/wp-module-ecommerce.pot --headers='{\"Report-Msgid-Bugs-To\":\"https://github.com/newfold-labs/wp-module-ecommerce/issues\",\"POT-Creation-Date\":null}' --exclude=node_modules,src,tests"
+        ],
+        "fix": [
+          "vendor/bin/phpcbf --standard=phpcs.xml ."
+        ],
+        "lint": [
+          "vendor/bin/phpcs --standard=phpcs.xml -s ."
+        ]
+      },
+    "config": {
+        "optimize-autoloader": true,
+        "sort-packages": true,
+        "platform": {
+          "php": "7.0.0"
```

Comment:
This should reflect our current lowest supported PHP version, which is currently 7.3. See https://github.com/newfold-labs/wp-plugin-hostgator/blob/main/.github/workflows/cypress-matrix.yml#L24